skiko/build.gradle.kts

```diff
@@ -1332,3 +1389,9 @@ rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJ
         nodeExtension.download = false
     }
 }
+
+project.tasks.withType<org.jetbrains.kotlin.gradle.dsl.KotlinJsCompile>().configureEach {
+    kotlinOptions.freeCompilerArgs += listOf(
+        "-Xwasm-enable-array-range-checks", "-Xir-dce=true"
```

Comment:
Does `-Xir-dce` enables DCE even for debug builds?
Do we need it for debug builds?

Reply:
This flag here affects only for tests I suppose. It does not matter to have it or not. I can remove it.